en/docs/learn/configuring-a-service-provider-for-adaptive-authentication.md

```diff
@@ -17,6 +17,24 @@ This tutorial guides you through setting up a sample application to try
 out adaptive authentication and configuring it as a service provider in
 WSO2 IS.
 
+!!! Warning "Before you proceed"
+
+    Adaptive Authentication is disabled by default.
+    You need to run **adaptive.sh** (**adaptive.bat** for Windows) in 
```

Comment:
The name of the script in the latest pack is **enableadaptive.sh**

Reply:
ACK